ci/build-ios.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e -u -o -x pipefail
+
+cd "$(dirname "$0")/../"
+
+# Get shared CI and prepare Unity
+ci/bootstrap.sh
+.shared-ci/scripts/prepare-unity.sh
+
+source ".shared-ci/scripts/pinned-tools.sh"
+
+if ! isMacOS; then
+    echo "iOS can only be built on macOS"
+    exit 0
```

Comment:
exit 1?

Reply:
Yup, mixed those up, thanks for pointing out!